bool all
string data
evd_ros_core/Version tag
---
bool status
string errors
string message
